src/components/AirTicketsSearch/AirTicketsSearch.component.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import DateRangePicker from 'react-bootstrap-daterangepicker';
+import 'bootstrap-daterangepicker/daterangepicker.css';
+
+import ItineraryModalContainer from '../ItinerarayModalContainer/ItineraryModalContainer.component';
+import PassengSubContent from '../PassengerSubContent/PassengerSubContent.component';
+
+import {
+	AirTicketsSearchContainer,
+	SearchContainer,
+} from './AirTicketsSearch.styles';
+
+const AirTicketsSearch = () => {
+	const [passengerSubContentHidden, setPassengerSubContentHidden] = useState(
+		true,
+	);
+	const [adultAmount, setAdultAmount] = useState(0);
+	const [childAmount, setChildAmount] = useState(0);
+	const [babyAmount, setBabyAmount] = useState(0);
+	const [departure, setDeparture] = useState('김포');
+	const [arrival, setArrival] = useState('제주');
+	const [modalHidden, setModalHidden] = useState(true);
+	const [seatSelection, setSeatSelection] = useState([
+		true,
+		false,
+		false,
+		false,
+	]);
+
+	const [locationTarget, setLocationTarget] = useState(null);
+
+	const changeLocation = (e) => {
+		if (locationTarget === departure) setDeparture(e.target.innerText);
+		else setArrival(e.target.innerText);
+		setModalHidden(!modalHidden);
+	};
+
+	const getSelection = (num) => {
+		let selectionArray = [false, false, false, false];
+		selectionArray[num] = true;
+		setSeatSelection(selectionArray);
+	};
+
+	const changeTargetAmount = (increase, targetAmount, targetFunction) => {
+		if (targetAmount === 0 && !increase) return;
+		increase
+			? targetFunction(targetAmount + 1)
+			: targetFunction(targetAmount - 1);
+	};
+
+	const totalAmount = () => {
+		return adultAmount + childAmount + babyAmount;
+	};
+
+	const checkSearchAvailability = () => {
+		!totalAmount() && alert('인원 수 문제');
+	};
+
+	const swapDepartureAndArrival = () => {
+		const temp = departure;
+		setDeparture(arrival);
+		setArrival(temp);
+	};
```

Comment:
```suggestion
	const swapDepartureAndArrival = () => {
		setDeparture(arrival);
		setArrival(departure);
	};
```
굳이 한번 더 변수로 선언하는 이유가 없어보입니다

Reply:
action 함수여서 첫 번째 함수가 실행되고 두 번째 함수의 인자 값이 그 영향을 받아서 temp에 넣어 놨습니다! 😄